src/pages/workspace/WorkspaceCardPage.js

```diff
@@ -110,6 +110,8 @@ const WorkspaceCardPage = ({
                 onCloseButtonPress={() => Navigation.dismissModal()}
                 onBackButtonPress={() => Navigation.goBack()}
                 shouldShowBackButton={isSmallScreenWidth}
+                shouldShowInboxCallButton
+                inboxCallTaskID="WorkspaceCompanyCards"
```

Comment:
Should we make a const for this?

Reply:
I don't think it needs it, we are only using it in a single location. If it was used in more locations I would agree with you